### PR TITLE
Update .net Framework 4.8

### PR DESCRIPTION
**Pull request by Yako0b**
Updated project and dependecies to .net Framework 4.8 to make is usable with VS 2022.

### CODE REVIEW

File: Log4NetWebViewer/Services/WebService.cs [hunk 1]
```diff
@@ -31,6 +31,8 @@ public void Dispose()
 
     public class CustomConventionsBootstrapper : DefaultNancyBootstrapper
     {
+
```
Comment (GKotfis) [inline Log4NetWebViewer/Services/WebService.cs]: Please remove unnecessary empty lines

**Review by GKotfis — CHANGES_REQUESTED** [review]
Just requested one small changes (remove empty lines). Everything else looks good to me 👍

**Comment by Yako0b** [comment]
Empty lines were removed!

**Review by GKotfis — APPROVED** [review]

**Comment by GKotfis** [comment]
Thanks @Yako0b 🎉

**Pull request merged**